index.bs

```diff
@@ -2435,12 +2462,18 @@ To <dfn>match [=filter values=] with negation</dfn> given a [=filter value=] |a|
 1. If |i| is not [=set/is empty|empty=], then return false.
 1. Return true.
 
-To <dfn>match an attribution source's filter data against a filter map</dfn> given an
-[=attribution source=] |source|, a [=filter map=] |filter|, and a [=boolean=]
-<dfn for="match an attribution source's filter data against a filter map"><var>isNegated</var></dfn>:
+To <dfn>match an attribution source's filter data against a filter config</dfn> given an
+[=attribution source=] |source|, a [=filter config=] |filter|, a [=moment=] |moment|, and a [=boolean=]
+<dfn for="match an attribution source's filter data against a filter config"><var>isNegated</var></dfn>:
+
+1. Let |lookbackWindow| be |filter|'s [=filter config/lookback window=].
+1. If |lookbackWindow| is not null:
+    1. If |moment| - |source|'s [=attribution source/source time=] is greater than |lookbackWindow|:
```

Comment:
Let's use https://w3c.github.io/hr-time/#dfn-duration-from, as we do elsewhere in the spec, instead of `-` here.